configs/ssg_platform_tools-vfxplatform-eln.yaml

```diff
@@ -0,0 +1,50 @@
+document: feedback-pipeline-workload
```

Comment:
Please rename this file to `sst_pt_libraries-vfxplatform-eln.yaml` since we also want to avoid SSG Platform Tools in the names. I have an outstanding MR to clean this up here: https://github.com/minimization/content-resolver-input/pull/939